File: type-ui/src/main/java/dev/sayaya/handbook/client/domain/validator/ValidatorDate.java
package dev.sayaya.handbook.client.domain.validator;

import lombok.Builder;
import lombok.Data;
import lombok.experimental.Accessors;

@Data
@Accessors(fluent = true)
@Builder
public class ValidatorDate implements ValidatorDefinition {
    private final Double min;
    private final Double max;
}

File: entity/src/main/kotlin/dev/sayaya/handbook/entity/validator/ValidatorDate.kt
package dev.sayaya.handbook.entity.validator

import java.time.Instant

data class ValidatorDate (
    val before: Instant? = null, // 이 날짜 이전이어야 함
    val after: Instant? = null   // 이 날짜 이후여야 함
) : ValidatorDefinition {
    override val type: ValidatorType get() = ValidatorType.DATE
}
